docs/docs/api/02-Commands/expect.md

```diff
@@ -3,7 +3,9 @@ displayed_sidebar: apiSidebar
 ---
 # expect
 
-Expects a condition, e.g., `exists()` or `notExits()`.
+Throws an error when an element specified by a filter `exists` or `notExists`.
+
+Use the structure `expect().<your filter>.(exists()|notExists()` as shown in the examples below.
```

Comment:
A closing parenthesis is missing at the end:
 `expect().<your filter>.(exists()|notExists()`